### DISCUSSION TITLE
Creating a Continuous Contig from SPAde Output

### DISCUSSION THREAD

**Discussion by Abrar-co** [ablab/spades#1329] [Q&A]
Hello everyone, I'm encountering an issue with the output file from the SPAde. The contigs.fasta file contains numerous contigs. How can I merge these contigs into a single continuous contig or assemble them into a complete genome sequence?

**Comment by asl** [comment]
It is not possible in general. Lots of things contributes to contiguity of your assembly including the genome properties, your input data, etc. There is vast literature on this subject, I would suggest you to familiarize with it.
